atom/common/api/atom_api_crash_reporter.cc

```diff
@@ -31,17 +33,39 @@ struct Converter<std::map<std::string, std::string> > {
   }
 };
 
+template<>
+struct Converter<std::vector<CrashReporter::UploadReportResult> > {
+  static v8::Local<v8::Value> ToV8(v8::Isolate* isolate,
+      const std::vector<CrashReporter::UploadReportResult>& reports) {
+    v8::Local<v8::Array> result(v8::Array::New(isolate, reports.size()));
+    for (size_t i = 0; i < reports.size(); ++i) {
+      mate::Dictionary dict(isolate, v8::Object::New(isolate));
+      dict.Set("date", reports[i].first);
```

Comment:
How to convert the integer to JS Date here? I can't find any code sample in the repo.